### PR TITLE
Let lesson edit page use the full width of the screen

### PR DESCRIPTION
**Pull request by davidsbailey**
Finishes [PLAT-343]. Because only the activities editor needs the full width, I've only expanded the width of that section. 

### before

Screen Shot 2020-10-20 at 9 44 39 AM

### after

wide-lesson

wide-lesson-responsive

## Testing story

New unit test for CollapsibleEditorSection fullWidth param. existing test coverage verifies the page still renders.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

[PLAT-343]: https://codedotorg.atlassian.net/browse/PLAT-343

### CODE REVIEW

**Comment by davidsbailey** [comment]
@tess323 please let me know if you have any input on the screenshots. the floating Save button change is not captured here yet.

**Comment by dmcavoy** [comment]
@davidsbailey can you make the CollapsibleEditorSections take up the full width?

**Comment by dmcavoy** [comment]
Ah sorry didn't look in the code. Looks like you intentional didn't make all the editors fill the whole screen. Why?

**Comment by davidsbailey** [comment]
@dmcavoy some of the inputs look a bit wide without any restriction on their width:
Screen Shot 2020-10-20 at 11 22 08 AM

I am open to alternatives :-)

**Comment by dmcavoy** [comment]
@davidsbailey I see what you are saying. I was responding to the fact that the header doesn't extend the full width so maybe the header should go the full width and then we limit the width of components within the header? Also It might be nice to move the markdown previews to the right instead of underneath to better use the space. But that can be done in a follow up.

**Comment by davidsbailey** [comment]
> maybe the header should go the full width and then we limit the width of components within the header?

great idea, this definitely looks better to me:

only-narrow-content

I've updated the PR to reflect this.

**Comment by davidsbailey** [comment]
> Also It might be nice to move the markdown previews to the right instead of underneath to better use the space.

Is this better? 

localhost-studio code org_3000_lessons_6094_edit (1)

I have not yet updated the code in the PR to reflect this image.

**Comment by tess323** [comment]
I think left aligned is a bit more UI friendly particularly on bigger screens.  If it is easy, I echo Dani's idea about moving previews for overview fields to the right on larger screens.  If it is difficult I think it is fine to have it below.

**Comment by dmcavoy** [comment]
+1 for left aligned

**Comment by davidsbailey** [comment]
sorry, @dmcavoy , I totally misunderstood your request to move preview columns over for markdown fields. I will give that a try.

**Comment by davidsbailey** [comment]
OK I got a bit carried away here with making it responsive but am pretty happy with how it ended up!

## full screen

localhost-studio code org_3000_lessons_6094_edit (3)

## responsive

wide-markdown

File: apps/src/lib/levelbuilder/TextareaWithMarkdownPreview.jsx [hunk 1]
```diff
@@ -4,10 +4,24 @@ import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
...
+    padding: 5,
+    display: 'flex',
+    alignItems: 'flex-start',
+    flexWrap: 'wrap'
+  },
+  container: {
+    flex: '1 1 500px',
+    maxWidth: 970,
+    margin: 5
+  },
```
Comment (davidsbailey) [inline apps/src/lib/levelbuilder/TextareaWithMarkdownPreview.jsx:20]: I definitely had to look up how to do this: https://stackoverflow.com/a/26129661/3991031

**Review by dmcavoy — APPROVED** [review]
Great work!

**Pull request merged**